common/src/main/java/com/wynntils/models/players/type/GuildInfo.java

```diff
@@ -21,11 +26,13 @@ public record GuildInfo(
         int xpPercent,
         int territories,
         long wars,
-        String createdTimestamp,
+        Date createdTimestamp,
         int totalMembers,
         int onlineMembers,
         List<GuildMemberInfo> guildMembers) {
     public static class GuildDeserializer implements JsonDeserializer<GuildInfo> {
+        private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.ROOT);
```

Comment:
The locale isn't relating to timezone here

> locale – the locale whose date format symbols should be used

Reply:
Ok. I got a bit confused there. Anyway, here is a better way to do this:
https://stackoverflow.com/questions/6543174/how-can-i-parse-utc-date-time-string-into-something-more-readable

Reply:
You should return this as an `Instant`, not a `Date`. An `Instant` basically means "timestamp with suitable metadata" in java.time. The second major component is `Duration`, which describes a period of time between two instants.

I guess we have been a bit sloppy with using these properly, but we really really should be using only the new java.time API. I would even like to add a code check rule for Wynntils that bans java.util.Date. It is just bad.

Reply:
Alright, I'll see if I can fix up all the places that uses it currently, it's only a few places

Reply:
I have a vague memory about some comment claiming that we needed to use an outdated `Date` construct due to one reason or another. I'm not sure that reason is valid (it can be that it just needs some more effort to convert to a proper API call), but maybe it is -- if so, you can keep it like that. But if you can fix it too, that'd be great. `<insert meme here>`

Reply:
How's https://github.com/Wynntils/Artemis/pull/2517/commits/203fe2752c261954ed365d2e903cf47de936e57b? If that looks good I'll apply that elsewhere

Reply:
LGTM